python-avd/pyavd/_eos_designs/structured_config/base/__init__.py

```diff
@@ -425,8 +428,24 @@ def local_users(self) -> None:
         """local_users set based on global aaa_settings.local_users data model."""
         if not (local_users := self.inputs.aaa_settings.local_users):
             return
-
-        self.structured_config.local_users = local_users._natural_sorted()
+        for local_user in local_users._natural_sorted():
+            local_user_data = EosCliConfigGen.LocalUsersItem(
+                name=local_user.name,
+                disabled=local_user.disabled,
```

Comment:
check if this is True and only render this in that case as everything else will be ignored (avoiding potential costly password computation as you mentiuonned on the call)